src/main/kotlin/simplerag/ragback/domain/prompt/entity/enums/PreSet.kt

```
@@ -0,0 +1,5 @@
+package simplerag.ragback.domain.prompt.entity.enums
+
+enum class PreSet {
+
+}
```

detekt (1.23.8)

[warning] The class or object PreSet is empty.

(detekt.empty-blocks.EmptyClassBlock)